database/migrations/2017_09_20_100733_create_request_headers_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRequestHeadersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('request_headers', function (Blueprint $table) {
+            $table->increments('id');
+
+            $table->integer('request_id')
+                ->unsigned();
+            $table->foreign('request_id')
+                ->references('id')->on('requests');
+
+            $table->string('name');
+            $table->string('value');
```

Comment:
Maybe `text()` or `mediumText()`, just in case?
👉  https://stackoverflow.com/questions/686217/maximum-on-http-header-values